index.html

```diff
@@ -48,186 +59,220 @@ <h3><a href="#">LET'S CONNECT</a></h3>
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="work-section4">
+                <img src="image-resources/Snapshoot Portfolio2.jpg" alt="">
+            </div>
+        </section>
+    
+        <section class="work-section">
+            <div class="work-section1">
+                <img src="image-resources/Portfolio.png" alt="">
+            </div>
+            <div class="work-section2">
+                <div class="work-imgs card3">
+                    <div class="work-bg"></div>
+                </div>
+                <div class="work-topic">
+                    <h2>Facebook 360</h2>
+                </div>
+                <div class="work-details">
+                    <ul>
+                        <li>Facebook</li>
+                        <li class="work-2nd">∘ Full Stack Dev</li>
+                        <li class="work-2nd">∘ 2015</li>
+                    </ul>
+                </div>
+                <div class="work-paragraph">
+                    <p>Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
+                </div>
+                <div class="work-langs">
+                     <ul>
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javascript</a></li>
+                    <li class="ruby"><a href="#">Ruby on Rails</a></li>
+                     </ul>
+                </div>
+                <div class="work-button">
+                    <button>See Project</button>
+                </div>
+            </div>
+        </section>
+    
+        <section class="work-section">
+            <div class="work-section3">
+                <div class="work-imgs card4">
+                    <div class="work-bg"></div>
+                </div>
+                <div class="work-topic">
+                    <h2>Uber Navigation</h2>
+                </div>
+                <div class="work-details">
+                    <ul>
+                        <li>Uber</li>
+                        <li class="work-2nd">∘ Lead Developer</li>
+                        <li class="work-2nd">∘ 2015</li>
+                    </ul>
+                </div>
+                <div class="work-paragraph">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="work-langs">
+                     <ul>
+                    <li><a href="#">html</a></li>
+                    <li class="ruby"><a href="#">Ruby on rails</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javascript</a></li>
+                     </ul>
+                </div>
+                <div class="work-button">
+                    <button>See Project</button>
```

Comment:
- [ OPTIONAL]I can see that you made a great design on this project but it could be better if your button `hover` once scroll on it as we can see that on the Figma design, kindly fix this for your design to be more attractive and professional :wink:
 I made this optional because it was not required in the past reviews but let's make look more professional by fixing this :point_down: 
 
![Screenshot from 2023-02-23 15-14-38](https://user-images.githubusercontent.com/88894525/220919338-8e4d67c1-5670-4bba-9da9-d832d1226022.png)